hdtv/database/PGAALibraries.py

```diff
@@ -63,13 +69,10 @@
 
     def __init__(
         self,
-        csvfile=os.path.join(
-            hdtv.datadir,
-            "PGAAlib-IKI2000.dat"),
+        csvfile=os.path.join(hdtv.datadir, "PGAAlib-IKI2000.dat"),
```

Comment:
Codacy found an issue: ['os' may be undefined, or defined from star imports: hdtv.database.common (F405)](https://app.codacy.com/manual/janmayer/hdtv/pullRequest?prid=6293112)